move_group/src/default_capabilities/move_action_capability.cpp

```diff
@@ -141,6 +141,7 @@ void move_group::MoveGroupMoveAction::executeMoveCallback_PlanOnly(const moveit_
 {
   ROS_INFO("Planning request received for MoveGroup action. Forwarding to planning pipeline.");
 
+  context_->planning_scene_monitor_->syncSceneUpdates();
```

Comment:
I believe we only have to sync here if `goal->request.start_state` is empty.
Otherwise we would sync our robot state even if we plan with totally unrelated start state.